pyspark/executors/requirements.txt

```diff
@@ -121,4 +121,5 @@ werkzeug==3.0.1 ; python_version >= "3.11" and python_version < "3.12"
 wheel==0.42.0 ; python_version >= "3.11" and python_version < "3.12"
 wrapt==1.14.1 ; python_version >= "3.11" and python_version < "3.12"
 xarray==2023.12.0 ; python_version >= "3.11" and python_version < "3.12"
+xgboost=3.0.0; python_version >= "3.11" and python_version < "3.12"
```

Comment:
and why not add the same to jupyterhub image too so there is no version mismatch?